scalableminds/webknossos
# Update deprecated antd <Menu>

We regularly have depreacation warning in the browser dev tools that antd's `<Menu>{children}</Menu>` HOC has been replaced with an `items` prop: `<Menu items={...}>`. 

The recommend upgrade path from antd v4 to v5 is to fix all these deprecation warnings. In this pull request I fixed the `<Navbar>` and `<FolderTree>` menus.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Click on all navbar menu items. to make sure that they still work
- Consider styling changes (menu should look the same)

### Issues:
- fixes #6812
- Contributes to #6861

------
(Please delete unneeded items, merge only when none are left open)
